.github/workflows/ci.yml

```
@@ -400,7 +400,9 @@ jobs:
       - name: Run E2E dice ladder tests
         run: pytest tests_e2e/test_dice_ladder_e2e.py --no-cov -v
 
+  # Temporarily disabled during production-performance work. Restore via #679.
   test-e2e-playwright:
+    if: ${{ false }}
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)